config/config.go

```diff
@@ -58,12 +58,12 @@ type Config struct {
 	// Max length is 64, see Identifier Length Limits
 	// https://dev.mysql.com/doc/refman/8.0/en/identifier-length.html
 	KeyColumn string `key:"keyColumn" validate:"required,max=64"`
+	// Keyspace is a keyspace.
```

Comment:
could you please add some additional info in comment 
for example :
Keyspace  - Vittes keyspace

Reply:
Done